cmd/kubectl-kubestellar/cmd/kubectlKubestellar.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
We conventionally eschew uppercase in filenames because developers work on a variety of OSes, which have an unfortunate variety in the ways they handle upper case in filenames.  "kubectl-kubestellar.go" would be commonly used in place of "kubectlKubestellar".

Reply:
updated